Source/WebKit/UIProcess/WebAuthentication/Cocoa/AuthenticationServicesForwardDeclarations.h

```diff
@@ -332,11 +332,13 @@ typedef NSString *ASAuthorizationPublicKeyCredentialResidentKeyPreference;
 @end
 
 @interface ASAuthorizationSecurityKeyPublicKeyCredentialAssertion : NSObject <ASAuthorizationPublicKeyCredentialAssertion>
+@property (nonatomic, readonly) BOOL appID;
```

Comment:
It's confusing that this `BOOL` is named `appID`, while the related class also has a member named `appID` that is an `NSString *`. They clearly don't really refer to the same thing.

Should the `BOOL` member be named something a bit clearer, like 'hasAppID` or `shouldSupplyAppID` or something more meaningful?

Reply:
I agree that it's confusing. The naming for the extension input and output are defined in the [WebAuthn IDL here](https://www.w3.org/TR/webauthn-3/#sctn-appid-extension) and we generally want to match the spec for names in system APIs.

Reply:
That is an EXCELLENT reason to keep your naming scheme. Please disregard my suggestion! :-)